packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -329,13 +329,14 @@ export function renderTextPrompt(props: Omit<TextPromptProps, 'onSubmit'>): Prom
 }
 
 interface RenderTextOptions {
+  text: string
```

Comment:
🙃 This is a breaking change. It's ok to ship it because we are pinning the exact version, but we'll soon need to be careful about it.

Reply:
This API was just released so I'm pretty confident none is using it :) Also it can be replaced with `outputInfo` so I think that's ok.